Airports/EGCN/VRPs.txt

```diff
@@ -3,7 +3,7 @@
 *Haxey:N053.29.25.000:W000.50.37.000
 *Wadworth M18/A1M:N053.28.45.000:W001.08.54.000
 *Gainsborough:N053.23.34.000:W000.46.15.000
-*Thurcroft:N053.23.34.000:W001.16.43.000
-*Clumber:N053.17.47.000:W001.01.56.000
+*Thurcroft Int.:N053.23.34.000:W001.16.43.000
+*Clumber Int.:N053.17.47.000:W001.01.56.000
```

Comment:
I would suggest maybe making this match the naming convention with A1/A57 bit in, see other comment on EGNE's definition.